src/main/connection_listener/mod.rs

```diff
@@ -31,6 +30,8 @@ use std::sync::{Arc, Mutex};
 const LISTENER_BACKLOG: i32 = 100;
 
 /// Accepts connections and transitions each connection into `ExchangeMsg` state.
+/// Optionally will make `ExchangeMsg` to test for peer external reachability. This behavior
+/// is disabled by default.
```

Comment:
But it seems you have enabled it by default [below at line 168 in this file in this diff](https://github.com/maidsafe/crust/pull/1104/files#diff-99e3273d6f63d6af85f9577cc6dcfc54R168)

Reply:
oops, good catch :+1: Just to clarify, it's intended to be enabled by default, so I will update the docs :)